static/js/containers/ChannelPage_test.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import { assert } from 'chai'
+import sinon from 'sinon'
+
+import { makeChannelList } from '../factories/channels'
+import { makeChannelPostList } from '../factories/posts'
+import IntegrationTestHelper from '../util/integration_test_helper'
+import { actions } from '../actions'
+import { channelURL } from '../lib/url'
+
+describe('ChannelPage', () => {
+  let helper, renderComponent, listenForActions, currentChannel, otherChannel, channels, posts
+
+  beforeEach(() => {
+    channels = makeChannelList()
```

Comment:
I think it would be good to add an argument to `makeChannelList` for the number of channels to create, default it to its current value, and call `makeChannelList(5)` et al to make it explicit that we're creating 5 or more and that it's safe to reference index 4 in the resulting array